sys/riscv/riscv/pmap.c

```diff
@@ -3483,9 +3535,13 @@ pmap_copy_page(vm_page_t msrc, vm_page_t mdst)
 void
 pmap_copy_page_tags(vm_page_t msrc, vm_page_t mdst)
 {
+
+#if defined(INVARIANTS)
```

Comment:
I would probably just drop the #ifdef here.  We tend to not put #ifdef's around invocations of other assertion helpers (e.g. mtx_assert() and friends) by using #ifdef's in the definition if needed.